backend/app/usecase/search/search.go

```diff
@@ -38,50 +57,61 @@ func (s Search) Search(query Query, filter Filter) (Result, error) {
 		go func() {
 			result, err := s.searchResource(filter.resources[i], orders[i], query, filter)
 			if err != nil {
-				// TODO(issue#865): Handle errors of Search API
 				s.logger.Error(err)
-				resultCh <- Result{}
+				resultCh <- Result{
+					Resources: ResourceResult{},
+					Err:       err,
+				}
 				return
 			}
-			resultCh <- result
+			resultCh <- Result{
+				Resources: result,
+				Err:       nil,
+			}
 		}()
 	}
 
 	timeout := time.After(s.timeout)
-	var results []Result
+	var results []ResourceResult
+	var resultErr error
```

Comment:
This is not needed based on the previous comment.

Reply:
I think this is still needed, because we would like to return only the first error that occurred. What do you think?